mlir/include/mlir/Dialect/Affine/Passes.h

```diff
@@ -31,6 +31,12 @@ std::unique_ptr<OperationPass<FuncOp>> createSimplifyAffineStructuresPass();
 std::unique_ptr<OperationPass<FuncOp>>
 createAffineLoopInvariantCodeMotionPass();
 
+/// Creates a pass replacing store with affine.store
+std::unique_ptr<OperationPass<FuncOp>> replaceAffineStorePass();
+
```

Comment:
Is there a reason for having these as separate passes rather than a single replace affine?

Reply:
No, just easier to debug. I will merge the two in the next commit.